pkg/restic/backupper.go

```diff
@@ -168,6 +168,10 @@ ForEachVolume:
 		case res := <-resultsChan:
 			switch res.Status.Phase {
 			case velerov1api.PodVolumeBackupPhaseCompleted:
+				if res.Status.SnapshotID == "" {
```

Comment:
let's add a comment here explaining that this would be the case if the volume was empty and therefore we don't actually have a restic snapshot, since it's not immediately obvious.